docs/ops/deployment/yarn_setup.md

```diff
@@ -132,7 +132,10 @@ Stop the YARN session by stopping the unix process (using CTRL+C) or by entering
 
 Flink on YARN will only start all requested containers if enough resources are available on the cluster. Most YARN schedulers account for the requested memory of the containers,
 some account also for the number of vcores. By default, the number of vcores is equal to the processing slots (`-s`) argument. The `yarn.containers.vcores` allows overwriting the
-number of vcores with a custom value.
+number of vcores with a custom value. In order for this parameter to be used your cluster must have CPU scheduling enabled. You can do this e.g. by
+
+ * setting the `org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler` or
+ * enabling `org.apache.hadoop.yarn.util.resource.DominantResourceCalculator` for `org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler`
```

Comment:
I would suggest adding this to the options documentation, and then simply link to the configuration section. (you can link to individual options!)

Reply:
Changed it as you suggested.